--- shell/eog-window.c.orig	2005-10-03 18:04:33.000000000 -0300
+++ shell/eog-window.c	2005-10-14 11:32:51.000000000 -0300
@@ -870,7 +870,7 @@
 	if (err_code == EOG_IMAGE_ERROR_FILE_EXISTS) {
 		char *str;
 
-		str = eog_image_get_uri_for_display (edata->image); 
+		str = gnome_vfs_uri_to_string (data->target->uri, GNOME_VFS_URI_HIDE_TOPLEVEL_METHOD);
 		header = g_strdup_printf (_("Overwrite file %s?"), str);
 		detail = _("File exists. Do you want to overwrite it?");
 		g_free (str);
