--- shell/eog-window.c	2004-11-29 18:53:13.000000000 -0200
+++ shell/eog-window.c.new	2005-08-10 19:44:00.243719616 -0300
@@ -642,7 +642,7 @@
 	if (err_code == EOG_IMAGE_ERROR_FILE_EXISTS) {
 		char *str;
 
-		str = eog_image_get_uri_for_display (image); 
+		str = gnome_vfs_uri_to_string (data->target->uri, GNOME_VFS_URI_HIDE_TOPLEVEL_METHOD);
 
 		header = g_strdup_printf (_("Overwrite file %s?"), str);
 		detail = _("File exists. Do you want to overwrite it?");
