diff -Nur eog-2.16.2/shell/main.c eog-2.16.2.new/shell/main.c
--- eog-2.16.2/shell/main.c	2007-02-27 15:08:41.000000000 +0100
+++ eog-2.16.2.new/shell/main.c	2007-02-27 15:09:45.000000000 +0100
@@ -413,6 +413,9 @@
 static void
 job_prepare_model_finished (EogJob *job, gpointer data, GError *error)
 {
+	GtkWidget *dialog;
+	EogImage *image;
+	int pos;
 	LoadContext *ctx = (LoadContext*) data;
 	
 	if (eog_job_get_status (job) == EOG_JOB_STATUS_FINISHED &&
@@ -423,8 +426,25 @@
 		assign_model_to_window (ctx->window, ctx->img_list);
 	} 
 	else {
+		pos = eog_image_list_get_initial_pos (ctx->img_list);
+		image = eog_image_list_get_img_by_pos (ctx->img_list, pos);
+		dialog = gtk_message_dialog_new (NULL,
+						 GTK_DIALOG_DESTROY_WITH_PARENT,
+						 GTK_MESSAGE_ERROR,
+						 GTK_BUTTONS_CLOSE,
+						 _("Image loading failed for %s"),
+						 eog_image_get_caption (image));
+		gtk_message_dialog_format_secondary_text (GTK_MESSAGE_DIALOG (dialog),
+							  error->message);
+		gtk_dialog_run (GTK_DIALOG (dialog));
+		gtk_widget_destroy (dialog);
+		
+		g_object_unref (image);
 		g_error_free (error);
-		gtk_main_quit ();
+
+		if (eog_get_window_list () == NULL) {
+			gtk_main_quit ();
+		}
 	}
 }
 
