diff --git a/data/fullscreen-toolbar.ui b/data/fullscreen-toolbar.ui
index e09570ee..2680d952 100644
--- a/data/fullscreen-toolbar.ui
+++ b/data/fullscreen-toolbar.ui
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Generated with glade 3.18.1 -->
+<!-- Generated with glade 3.22.1 -->
 <interface>
   <requires lib="gtk+" version="3.14"/>
   <object class="GtkToolbar" id="fullscreen_toolbar">
@@ -121,21 +121,6 @@
         <property name="homogeneous">True</property>
       </packing>
     </child>
-    <child>
-      <object class="GtkToggleToolButton" id="zoom_fit_toggle">
-        <property name="visible">True</property>
-        <property name="can_focus">False</property>
-        <property name="tooltip_text" translatable="yes">Fit the image into window</property>
-        <property name="action_name">win.toggle-zoom-fit</property>
-        <property name="label" translatable="yes">_Best Fit</property>
-        <property name="use_underline">True</property>
-        <property name="icon_name">zoom-fit-best-symbolic</property>
-      </object>
-      <packing>
-        <property name="expand">False</property>
-        <property name="homogeneous">True</property>
-      </packing>
-    </child>
     <child>
       <object class="GtkSeparatorToolItem" id="separator2">
         <property name="visible">True</property>
