diff --git a/src/eog-properties-dialog.c b/src/eog-properties-dialog.c
index 07507f01..30317407 100644
--- a/src/eog-properties-dialog.c
+++ b/src/eog-properties-dialog.c
@@ -202,7 +202,7 @@ pd_update_general_tab (EogPropertiesDialog *prop_dlg,
 	gtk_widget_set_sensitive (prop_dlg->priv->folder_button, FALSE);
 	gtk_button_set_label (GTK_BUTTON (prop_dlg->priv->folder_button), NULL);
 	g_free (prop_dlg->priv->folder_button_uri);
-	prop_dlg->priv->folder_button_uri = g_file_get_uri (parent_file);
+	prop_dlg->priv->folder_button_uri = g_file_get_uri (file);
 
 	g_file_query_info_async (parent_file,
 				 G_FILE_ATTRIBUTE_STANDARD_DISPLAY_NAME,
@@ -438,17 +438,19 @@ pd_exif_details_activated_cb (GtkExpander *expander,
 static void
 pd_folder_button_clicked_cb (GtkButton *button, gpointer data)
 {
-	EogPropertiesDialogPrivate *priv = EOG_PROPERTIES_DIALOG (data)->priv;
-	GtkWindow *window;
-	guint32 timestamp;
+	EogPropertiesDialogPrivate *priv = EOG_PROPERTIES_DIALOG(data)->priv;
+    GFile *file;
+    GtkWindow *window;
 
 	if (!priv->folder_button_uri)
 		return;
 	
-	timestamp = gtk_get_current_event_time ();
+    file = g_file_new_for_uri (priv->folder_button_uri);
+    
+    window = GTK_WINDOW(data);
+    eog_util_show_file_in_filemanager (file, window);
 
-	window = GTK_WINDOW (data);
-	gtk_show_uri_on_window (window, priv->folder_button_uri, timestamp, NULL);
+    g_object_unref(file);
 }
 
 static gboolean
