diff --git a/src/eog-file-chooser.c b/src/eog-file-chooser.c
index 03052549..64913fe4 100644
--- a/src/eog-file-chooser.c
+++ b/src/eog-file-chooser.c
@@ -356,13 +356,11 @@ update_preview_cb (GtkFileChooser *file_chooser, gpointer data)
 		if (thumb_path != NULL && g_file_test (thumb_path, G_FILE_TEST_EXISTS)) {
 			/* try to load and display preview thumbnail */
 			pixbuf = gdk_pixbuf_new_from_file (thumb_path, NULL);
-		} else if (g_file_info_get_size (file_info) <= 100000) {
+		} else {
 			/* read files smaller than 100kb directly */
-
 			gchar *mime_type = g_content_type_get_mime_type (
 						g_file_info_get_content_type (file_info));
 
-
 			if (G_LIKELY (mime_type)) {
 				gboolean can_thumbnail, has_failed;
 
@@ -422,6 +420,7 @@ eog_file_chooser_add_preview (GtkWidget *widget)
 	priv->size_label = gtk_label_new (NULL);
 	priv->creator_label = gtk_label_new (NULL);
 
+
 	gtk_box_pack_start (GTK_BOX (vbox), priv->image, FALSE, TRUE, 0);
 	gtk_box_pack_start (GTK_BOX (vbox), priv->dim_label, FALSE, TRUE, 0);
 	gtk_box_pack_start (GTK_BOX (vbox), priv->size_label, FALSE, TRUE, 0);
