diff --git a/src/eog-window.c b/src/eog-window.c
index 4e913271..6b980fd0 100644
--- a/src/eog-window.c
+++ b/src/eog-window.c
@@ -2299,8 +2299,19 @@ eog_window_action_file_open (GSimpleAction *action,
 					EOG_CONF_UI_FILECHOOSER_XDG_FALLBACK);
 		pics_dir = g_get_user_special_dir (G_USER_DIRECTORY_PICTURES);
 		if (use_fallback && pics_dir) {
+            GDir *dir;
+            GError *error;
+
 			gtk_file_chooser_set_current_folder (GTK_FILE_CHOOSER (dlg),
 							     pics_dir);
+
+            dir = g_dir_open (pics_dir, 0, &error);
+            if(g_dir_read_name(dir) == NULL){
+                GSettings *chooser_settings = g_settings_new ("org.gtk.Settings.FileChooser");
+                GVariant *sort_modified = g_variant_new_string ("modified");
+                g_settings_set_value (chooser_settings, "sort-column", sort_modified);
+            }
+            g_dir_close(dir);
 		}
 	}
 
