diff --git a/data/popup-menus.ui b/data/popup-menus.ui
index aae780c0..2a713ae5 100644
--- a/data/popup-menus.ui
+++ b/data/popup-menus.ui
@@ -1,13 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <interface>
   <menu id="view-popup-menu">
-    <section>
-      <submenu>
-        <attribute name="label" translatable="yes">Open _with</attribute>
-        <link name="submenu" id="open-with-menu">
-        </link>
-      </submenu>
-    </section>
     <section>
       <item>
         <attribute name="label" translatable="yes">_Save</attribute>
diff --git a/po/ko.po b/po/ko.po
index 271a5872..bae8a51b 100644
--- a/po/ko.po
+++ b/po/ko.po
@@ -18,7 +18,7 @@ msgstr ""
 "Report-Msgid-Bugs-To: https://bugzilla.gnome.org/enter_bug.cgi?"
 "product=eog&keywords=I18N+L10N&component=general\n"
 "POT-Creation-Date: 2017-06-28 17:37+0000\n"
-"PO-Revision-Date: 2017-08-27 11:36+0900\n"
+"PO-Revision-Date: 2020-04-24 15:36+0900\n"
 "Last-Translator: Jeeyong Um <conr2d@gmail.com>\n"
 "Language-Team: GNOME Korea <gnome-kr@googlegroups.com>\n"
 "Language: ko\n"
@@ -26,7 +26,7 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=1; plural=0;\n"
-"X-Generator: Poedit 2.0.3\n"
+"X-Generator: Poedit 2.2.1\n"
 
 #: data/eog-app-menu.ui:7
 msgid "Prefere_nces"
@@ -919,10 +919,6 @@ msgstr ""
 "사용중인 플러그인 목록. 사용중인 플러그인의 “위치”는 포함하지 않습니다. 주어"
 "진 플러그인의 “위치”는 .eog-plugin 파일을 참고하십시오."
 
-#: data/popup-menus.ui:6
-msgid "Open _with"
-msgstr "다른 프로그램으로 열기(_W)"
-
 #: data/popup-menus.ui:29
 msgid "_Copy"
 msgstr "복사(_C)"
@@ -1410,6 +1406,10 @@ msgstr ""
 "“%s” 이미지를 외부 프로그램이 수정했습니다.\n"
 "다시 읽어들이시겠습니까?"
 
+#: src/eog-winodw.c:1128
+msgid "Open _with"
+msgstr "다른 프로그램으로 열기"
+
 #. Translators: This string is displayed in the statusbar
 #. * while saving images. The tokens are from left to right:
 #. * - the original filename
diff --git a/src/eog-scroll-view.c b/src/eog-scroll-view.c
index 6e931204..17fea3f3 100644
--- a/src/eog-scroll-view.c
+++ b/src/eog-scroll-view.c
@@ -2811,7 +2811,7 @@ eog_scroll_view_set_popup (EogScrollView *view,
                            GtkMenu *menu)
 {
 	g_return_if_fail (EOG_IS_SCROLL_VIEW (view));
-	g_return_if_fail (view->priv->menu == NULL);
+	//g_return_if_fail (view->priv->menu == NULL);
 
 	view->priv->menu = g_object_ref (GTK_WIDGET (menu));
 
diff --git a/src/eog-window.c b/src/eog-window.c
index 9a8b0ef7..17397d45 100644
--- a/src/eog-window.c
+++ b/src/eog-window.c
@@ -196,6 +196,7 @@ static void fullscreen_set_timeout (EogWindow *window);
 static void fullscreen_clear_timeout (EogWindow *window);
 static void update_action_groups_state (EogWindow *window);
 static void eog_window_update_open_with_menu (EogWindow *window, EogImage *image);
+static void eog_window_update_popup (EogWindow *window);
 static void eog_window_list_store_image_added (GtkTreeModel *tree_model,
 					       GtkTreePath  *path,
 					       GtkTreeIter  *iter,
@@ -997,6 +998,7 @@ eog_window_display_image (EogWindow *window, EogImage *image)
 	update_status_bar (window);
 
 	eog_window_update_open_with_menu (window, image);
+	eog_window_update_popup (window);
 
 	file = eog_image_get_file (image);
 	g_idle_add_full (G_PRIORITY_LOW,
@@ -1139,6 +1141,33 @@ eog_window_update_open_with_menu (EogWindow *window, EogImage *image)
 	g_list_free (apps);
 }
 
+static void
+eog_window_update_popup (EogWindow *window)
+{
+	EogWindowPrivate	*priv;
+	GtkBuilder			*builder;
+	GObject				*builder_object;
+	GtkWidget			*popup_menu;
+
+	priv = window->priv;
+
+	builder = gtk_builder_new_from_resource ("/org/gnome/eog/ui/popup-menus.ui");
+	builder_object = gtk_builder_get_object (builder, "view-popup-menu");
+	popup_menu = gtk_menu_new_from_model (G_MENU_MODEL (builder_object));
+	
+	if(g_menu_model_get_n_items (G_MENU_MODEL(priv->open_with_menu)) > 0){
+		GMenuItem	*item;
+
+		item = g_menu_item_new(_("Open _with"),NULL);
+		g_menu_item_set_link(item,"submenu", G_MENU_MODEL(priv->open_with_menu));
+		g_menu_prepend_item(builder_object, item);
+	}
+	
+	eog_scroll_view_set_popup (EOG_SCROLL_VIEW (priv->view),
+			GTK_MENU (popup_menu));
+
+	g_object_unref (popup_menu);
+}
 static void
 eog_window_clear_load_job (EogWindow *window)
 {
@@ -4428,17 +4457,28 @@ eog_window_construct_ui (EogWindow *window)
 
 	builder = gtk_builder_new_from_resource ("/org/gnome/eog/ui/popup-menus.ui");
 	builder_object = gtk_builder_get_object (builder, "view-popup-menu");
+#if 0
 	GObject *open_with_menu = gtk_builder_get_object (builder, "open-with-menu");
 	g_menu_append_section (G_MENU (open_with_menu),
 			       NULL,
 			       G_MENU_MODEL (priv->open_with_menu));
+
 	popup_menu = gtk_menu_new_from_model (G_MENU_MODEL (builder_object));
 
+	GMenuItem *item;
+
+	item = g_menu_item_new("Open _with",NULL);
+	g_menu_item_set_link(item,"submenu",G_MENU_MODEL(priv->open_with_menu));
+	g_menu_prepend_item(builder_object, item);
+
 	eog_scroll_view_set_popup (EOG_SCROLL_VIEW (priv->view),
 				   GTK_MENU (popup_menu));
 
 	g_object_unref (popup_menu);
 
+#endif
+	eog_window_update_popup(window);
+
 	gtk_paned_pack1 (GTK_PANED (hpaned),
 			 priv->overlay,
 			 TRUE,
