diff --git a/src/eog-window.c b/src/eog-window.c
index 183daed8..2f6f680d 100644
--- a/src/eog-window.c
+++ b/src/eog-window.c
@@ -2009,16 +2009,13 @@ update_ui_visibility (EogWindow *window)
 	g_simple_action_set_state (G_SIMPLE_ACTION (action), g_variant_new_boolean (visible));
 	gtk_widget_set_visible (priv->statusbar, visible);
 
-	if (priv->status != EOG_WINDOW_STATUS_INIT) {
-		visible = g_settings_get_boolean (priv->ui_settings,
-						  EOG_CONF_UI_IMAGE_GALLERY);
-		visible &= gtk_widget_get_visible (priv->nav);
-		visible &= (priv->mode != EOG_WINDOW_MODE_SLIDESHOW);
-		action = g_action_map_lookup_action (G_ACTION_MAP (window), "view-gallery");
-		g_assert (action != NULL);
-		g_simple_action_set_state (G_SIMPLE_ACTION (action), g_variant_new_boolean (visible));
-		gtk_widget_set_visible (priv->nav, visible);
-	}
+	visible = g_settings_get_boolean (priv->ui_settings,
+						EOG_CONF_UI_IMAGE_GALLERY);
+	visible = visible && !fullscreen_mode;
+	action = g_action_map_lookup_action (G_ACTION_MAP (window), "view-gallery");
+	g_assert (action != NULL);
+	g_simple_action_set_state (G_SIMPLE_ACTION (action), g_variant_new_boolean (visible));
+	gtk_widget_set_visible (priv->nav, visible);
 
 	visible = g_settings_get_boolean (priv->ui_settings,
 					  EOG_CONF_UI_SIDEBAR);
