diff --git a/src/eog-window.c b/src/eog-window.c
index 2f6f680d..ec1051fb 100644
--- a/src/eog-window.c
+++ b/src/eog-window.c
@@ -120,20 +120,20 @@ struct _EogWindowPrivate {
 	GSettings           *view_settings;
 	GSettings           *lockdown_settings;
 
-        EogListStore        *store;
-        EogImage            *image;
+    EogListStore        *store;
+    EogImage            *image;
 	EogWindowMode        mode;
 	EogWindowStatus      status;
 
-        GtkWidget           *overlay;
-        GtkWidget           *box;
-        GtkWidget           *layout;
-        GtkWidget           *cbox;
-        GtkWidget           *view;
-        GtkWidget           *sidebar;
-        GtkWidget           *thumbview;
-        GtkWidget           *statusbar;
-        GtkWidget           *nav;
+    GtkWidget           *overlay;
+    GtkWidget           *box;
+    GtkWidget           *layout;
+    GtkWidget           *cbox;
+    GtkWidget           *view;
+    GtkWidget           *sidebar;
+    GtkWidget           *thumbview;
+    GtkWidget           *statusbar;
+    GtkWidget           *nav;
 	GtkWidget           *message_area;
 	GtkWidget           *properties_dlg;
 
@@ -145,37 +145,37 @@ struct _EogWindowPrivate {
 	GtkWidget           *fullscreen_popup;
 	GSource             *fullscreen_timeout_source;
 
-	gboolean             slideshow_loop;
-	gint                 slideshow_switch_timeout;
+	gboolean            slideshow_loop;
+	gint                slideshow_switch_timeout;
 	GSource             *slideshow_switch_source;
 
-	guint                fullscreen_idle_inhibit_cookie;
+	guint               fullscreen_idle_inhibit_cookie;
 
-        EogJob              *load_job;
-        EogJob              *transform_job;
+    EogJob              *load_job;
+    EogJob              *transform_job;
 	EogJob              *save_job;
 	GFile               *last_save_as_folder;
 	EogJob              *copy_job;
 
-        guint                image_info_message_cid;
-        guint                tip_message_cid;
-	guint                copy_file_cid;
+    guint               image_info_message_cid;
+    guint               tip_message_cid;
+	guint               copy_file_cid;
 
-        EogStartupFlags      flags;
+    EogStartupFlags     flags;
 	GSList              *file_list;
 
 	EogWindowGalleryPos  gallery_position;
 	gboolean             gallery_resizable;
 
-	gboolean	     save_disabled;
-	gboolean             needs_reload_confirmation;
+	gboolean	        save_disabled;
+	gboolean            needs_reload_confirmation;
 
 	GtkPageSetup        *page_setup;
 
 	PeasExtensionSet    *extensions;
 	GtkWidget           *button_gallery;
 #ifdef HAVE_LCMS
-        cmsHPROFILE         *display_profile;
+        cmsHPROFILE     *display_profile;
 #endif
 };
 
@@ -2243,14 +2243,13 @@ eog_window_print (EogWindow *window)
 
 	if (window->priv->page_setup != NULL)
 		page_setup = g_object_ref (window->priv->page_setup);
-	else
+	else 
 		page_setup = NULL;
 
 	print = eog_print_operation_new (window->priv->image,
 					 print_settings,
 					 page_setup);
 
-
 	// Disable page setup options if they are locked down
 	page_setup_disabled = g_settings_get_boolean (window->priv->lockdown_settings,
 						      EOG_CONF_DESKTOP_CAN_SETUP_PAGE);
@@ -2570,6 +2569,8 @@ eog_window_action_show_hide_bar (GSimpleAction *action,
 			g_settings_set_boolean (priv->ui_settings,
 						EOG_CONF_UI_STATUSBAR, visible);
 
+		gtk_widget_grab_focus (priv->view);
+
 	} else if (g_ascii_strcasecmp (g_action_get_name (G_ACTION (action)), "view-gallery") == 0) {
 		if (visible) {
 			/* Make sure the focus widget is realized to
@@ -2602,6 +2603,7 @@ eog_window_action_show_hide_bar (GSimpleAction *action,
 		g_simple_action_set_state (action, state);
 		g_settings_set_boolean (priv->ui_settings, EOG_CONF_UI_SIDEBAR,
 					visible);
+		gtk_widget_grab_focus (priv->view);
 	}
 }
 
