diff --git a/src/eog-sidebar.c b/src/eog-sidebar.c
index e07ddca7..a4f6effd 100644
--- a/src/eog-sidebar.c
+++ b/src/eog-sidebar.c
@@ -391,24 +391,16 @@ eog_sidebar_init (EogSidebar *eog_sidebar)
 	select_hbox = gtk_box_new (GTK_ORIENTATION_HORIZONTAL, 6);
 
 	eog_sidebar->priv->label = gtk_label_new ("");
-	gtk_widget_set_name (eog_sidebar->priv->label, "eog-sidebar-title");
-
-	gtk_box_pack_start (GTK_BOX (select_hbox),
-			    eog_sidebar->priv->label,
-			    FALSE, FALSE, 0);
-
-	gtk_widget_show (eog_sidebar->priv->label);
 
 	arrow = gtk_image_new_from_icon_name ("pan-down-symbolic", GTK_ICON_SIZE_BUTTON);
 	gtk_box_pack_end (GTK_BOX (select_hbox), arrow, FALSE, FALSE, 0);
 	eog_sidebar->priv->arrow = arrow;
 	gtk_widget_set_visible (arrow, FALSE);
 
-	gtk_container_add (GTK_CONTAINER (eog_sidebar->priv->select_button), select_hbox);
 	gtk_widget_show (select_hbox);
 
-	gtk_box_set_center_widget (GTK_BOX (hbox), eog_sidebar->priv->select_button);
-	gtk_widget_show (eog_sidebar->priv->select_button);
+	gtk_box_set_center_widget (GTK_BOX (hbox), eog_sidebar->priv->label);
+	gtk_widget_show (eog_sidebar->priv->label);
 
 	close_button = gtk_button_new ();
 
