diff --git a/src/eog-metadata-sidebar.c b/src/eog-metadata-sidebar.c
index b9131d13..a9a2c409 100644
--- a/src/eog-metadata-sidebar.c
+++ b/src/eog-metadata-sidebar.c
@@ -194,7 +194,27 @@ eog_metadata_sidebar_update_metadata_section (EogMetadataSidebar *sidebar)
 	ExifData *exif_data = NULL;
 
 	if (img) {
-		exif_data = eog_image_get_exif_info (img);
+        exif_data = eog_image_get_exif_info (img);
+        if(exif_data == NULL) {
+            gtk_label_set_selectable(GTK_LABEL (priv->aperture_label), false);
+            gtk_label_set_selectable(GTK_LABEL (priv->exposure_label), false);
+            gtk_label_set_selectable(GTK_LABEL (priv->focallen_label), false);
+            gtk_label_set_selectable(GTK_LABEL (priv->iso_label), false);
+            gtk_label_set_selectable(GTK_LABEL (priv->metering_label), false);
+            gtk_label_set_selectable(GTK_LABEL (priv->model_label), false);
+            gtk_label_set_selectable(GTK_LABEL (priv->date_label), false);
+            gtk_label_set_selectable(GTK_LABEL (priv->time_label), false);
+        }
+        else {
+            gtk_label_set_selectable(GTK_LABEL (priv->aperture_label), true);
+            gtk_label_set_selectable(GTK_LABEL (priv->exposure_label), true);
+            gtk_label_set_selectable(GTK_LABEL (priv->focallen_label), true);
+            gtk_label_set_selectable(GTK_LABEL (priv->iso_label), true);
+            gtk_label_set_selectable(GTK_LABEL (priv->metering_label), true);
+            gtk_label_set_selectable(GTK_LABEL (priv->model_label), true);
+            gtk_label_set_selectable(GTK_LABEL (priv->date_label), true);
+            gtk_label_set_selectable(GTK_LABEL (priv->time_label), true);
+        }
 	}
 
 	eog_exif_util_set_label_text (GTK_LABEL (priv->aperture_label),
