diff --git a/src/eog-preferences-dialog.c b/src/eog-preferences-dialog.c
index 7f9f3595..dc4cffd6 100644
--- a/src/eog-preferences-dialog.c
+++ b/src/eog-preferences-dialog.c
@@ -198,6 +198,7 @@ eog_preferences_dialog_init (EogPreferencesDialog *pref_dlg)
 {
 	EogPreferencesDialogPrivate *priv;
 	GtkAdjustment *scale_adjustment;
+    GList *list, *i;
 
 	pref_dlg->priv = eog_preferences_dialog_get_instance_private (pref_dlg);
 	priv = pref_dlg->priv;
@@ -303,6 +304,14 @@ eog_preferences_dialog_init (EogPreferencesDialog *pref_dlg)
 
 	gtk_widget_show_all (priv->plugin_manager);
 
+    list = gtk_container_get_children (priv->plugin_manager);
+    for(i = list; i!=NULL; i=i->next){
+        gchar *str = g_strdup(gtk_widget_get_name(i->data));
+
+        if(g_strcmp0(str,"GtkToolbar")== 0){
+            gtk_widget_hide(i->data);
+        }
+    }
 }
 
 GtkWidget *eog_preferences_dialog_get_instance(GtkWindow *parent)
